docs/src/examples/components/TextArea/Usage/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import ComponentExample from 'docs/src/components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/src/components/ComponentDoc/ExampleSection'
+
+const Usage = () => (
+  <ExampleSection title="">
```

Comment:
```suggestion
  <ExampleSection title="Usage">
```

Reply:
Actually, we don't specify `Usage`. As all the examples under Usage naturally belong to `Usage`.

Reply:
```suggestion
  <ExampleSection>
```

So it should be or even

```suggestion
  <UsageSection>
```

Otherwise it extremely confusing

Reply:
I agree its confusing 👍

Reply:
Let me create `UsageSection` in a seperate PR